### PR TITLE
Add configuration for new domain name iris.ac.uk

### PR DESCRIPTION
**Pull request by jose-caballero**
Add the public key and config file for brand new domain name iris.ac.uk
As this is only for the UK, and there is currently no repository with that domain name, it is fine for the time being to use only the Stratum-1 at RAL.

### CODE REVIEW

**Review by DrDaveD — CHANGES_REQUESTED** [review]

File: etc/cvmfs/domain.d/iris.ac.uk.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+. ../common.conf
```
Comment (DrDaveD) [inline etc/cvmfs/domain.d/iris.ac.uk.conf:1]: Please add the 4 lines of prefix before including common.conf that is in all the other domain.d/*.conf files.

**Review by DrDaveD — CHANGES_REQUESTED** [review]

**Comment by jose-caballero** [comment]
Fixed.

**Review by DrDaveD — APPROVED** [review]

**Comment by jose-caballero** [comment]
Should I issue the Merge myself, or someone else from the CVMFS core team have to do it?

**Comment by DrDaveD** [comment]
You do it, now that it is approved.

**Pull request merged**